client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -95,11 +115,12 @@ const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedC
 		setDetailsExpanded( Object.fromEntries( expandedEntries ) );
 	};
 
-	const setIntervalLength = ( interval: IntervalLength ) => {
+	const changeIntervalLength = ( interval: IntervalLength ) => {
 		if ( intervalLength === IntervalLength.ANNUALLY ) {
 			setDetailsExpanded( { google: false, titan: true } );
 		}
```

Comment:
Should we not continue to honour the `selectedEmailProviderSlug` in this logic?

Reply:
Very true. I think we should simply remove that condition.